cpp/src/barretenberg/stdlib/primitives/bigfield/bigfield.test.cpp

```diff
@@ -987,14 +934,14 @@ TYPED_TEST(stdlib_bigfield, division_context)
 //     auto prover = composer.create_prover();
```

Comment:
This commented out code wouldn't even build anymore if reinstated. Should it be removed?